recipes-core/initramfs-framework/files/0001-Mount-run-with-tmpfs.patch

```diff
@@ -0,0 +1,55 @@
+From 20d74a29c1c9050880f0456b2c3a0242c9c8ab70 Mon Sep 17 00:00:00 2001
+From: Sasi Kumar Maddineni <quic_sasikuma@quicinc.com>
+Date: Thu, 5 Jun 2025 15:02:40 +0530
+Subject: [PATCH] Mount /run with tmpfs
+
+Upstream-Status: Pending
+
+Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
+Git-Commit: 0c3b2340811774b495920c397c5603448446d160
+Git-repo: https://github.com/foundriesio/meta-lmp
+
+[Sasi Kumar Maddineni <quic_sasikuma@quicinc.com>: Resolved merge conflicts]
+Signed-off-by: Sasi Kumar Maddineni <quic_sasikuma@quicinc.com>
```

Comment:
We should try to propose this change upstream directly as well, but fine to have an initial patch here as it will also be needed with scarthgap.